README.md

```diff
@@ -36,6 +38,34 @@ The Javascript client library for <a href="https://interledger.org">Interledger<
 
 *Note that [ledger plugins](https://www.npmjs.com/search?q=ilp-plugin) must be installed alongside this module.*
 
+## [Loopback Transport (LT)](https://github.com/interledger/rfcs/pull/416)
+
+```js
+const PluginBtp = require('ilp-plugin-btp')
+const ILP = require('.')
+
+;(async function test () {
+  const pluginOut = new PluginBtp({ server: 'btp+wss://:plugin1@amundsen.ilpdemo.org:1810/' })
+  const pluginIn = new PluginBtp({ server: 'btp+wss://:plugin2@amundsen.ilpdemo.org:1810/' })
```

Comment:
should this use a plugin if we're expecting people to also use that BTP channel for transmitting data? The plugin doesn't expose any way to send other than `sendData`, which puts things into the ILP protocol data. So the full capability of BTP isn't exposed to the user.

The spec for LT also says that it uses BTP or HTTP-OER-ILP, but this implementation allows any plugin to be used. Maybe instead of taking a plugin for pluginIn, we should pass in a URL. A plugin may be constructed internally but the BTP connection could at least be exposed to the user. Unfortunately we'd need an API for that BTP connection because we can't expect the user to serialize messages manually.

Reply:
> should this use a plugin if we're expecting people to also use that BTP channel for transmitting data? 

This is just example code, it also has the amount `'10'` hard-coded. You could use the LT implementation from this PR using an augmented version of PluginBTP, which has an extra `sendCustomData` method. I do think, at least for now, we should implement LT here using LPIv2 plugins. If we see later that we want to create an LPIv3, we can always update this code again.

> this implementation allows any plugin to be used

Correct, because neither LT nor ILP put any restrictions on the ledger layer. 

> Maybe instead of taking a plugin for pluginIn, we should pass in a URL

Yes, but that would then be the payment pointer (i.e., the LSD URL). So you could do something like:

```js
const pluginOut = await ILP.LSD.setupLoopbackPlugin(`$example.com/bob`)
const loop = ILP.LT.createLoop({ pluginIn, pluginOut })
```

But this PR only implements that second step.